core/tests/puppeteer-acceptance-tests/blog-post-draft-blogs.js

```diff
@@ -0,0 +1,32 @@
+const acceptanceTests = require("./utility-functions/puppeteer_utils.js");
+const testConstants = require("./utility-functions/testConstants.js");
+
+
+const blogEditOptions = "e2e-test-blog-post-edit-box";
+
+async function deleteDraftAsBlogAdmin() {
+  const user = await new acceptanceTests();
```

Comment:
I think this doesn't quite make sense conceptually, a "user" is not an "acceptance test".

What is actually happening here -- what words/nouns would you use to describe it in plain English?

Reply:
behind the tests, there will be a test user that will check our expected behavior through the tests written! Vojta suggested this to name the class object as 'user'.